include/triton/core/tritonserver.h

```diff
@@ -847,6 +857,34 @@ TRITONSERVER_DECLSPEC TRITONSERVER_Error* TRITONSERVER_InferenceTraceParentId(
 TRITONSERVER_DECLSPEC TRITONSERVER_Error* TRITONSERVER_InferenceTraceModelName(
     TRITONSERVER_InferenceTrace* trace, const char** model_name);
 
+/// Set the name associated with a trace.
+///
+/// \param trace The trace.
+/// \param name The name associated with the trace.
+/// \return a TRITONSERVER_Error indicating success or failure.
+TRITONSERVER_DECLSPEC TRITONSERVER_Error*
+TRITONSERVER_InferenceTraceSetModelName(
+    TRITONSERVER_InferenceTrace* trace, const char* name);
```

Comment:
`model_name` to match with the Getter API
```suggestion
    TRITONSERVER_InferenceTrace* trace, const char* model_name);
```

Reply:
This API was added by mistake. I have removed it.